src/v2/view-builder/views/IdentifierView.js

```diff
@@ -160,6 +164,28 @@ const Body = BaseForm.extend({
     return newSchemas;
   },
 
+  showCustomFormErrorCallout(error) {
+    const {
+      errorSummaryKeys,
+      errorSummary,
+    } = error.responseJSON;
+
+    const isPossibleXss = errorSummary.includes('script') || errorSummary.includes('data');
```

Comment:
Extract to function and improve truthy conditions. This implementation may cause false positives/negatives.